src/southbridge/amd/pi/hudson/acpi/fch.asl

```diff
@@ -25,16 +25,17 @@
 Method(_OSC,4)
 {
 	// Create DWord-addressable fields from the Capabilities Buffer
-	CreateDWordField(Arg3,0,CDW1)
-	CreateDWordField(Arg3,4,CDW2)
-	CreateDWordField(Arg3,8,CDW3)
+	//CreateDWordField(Arg3,0,CDW1)
+	//CreateDWordField(Arg3,4,CDW2)
+	//CreateDWordField(Arg3,8,CDW3)
 
 	/* Check for proper PCI/PCIe UUID */
 	If(LEqual(Arg0,ToUUID("33DB4D5B-1FF7-401C-9657-7441C03DD766")))
 	{
 		/* Let OS control everything */
 		Return (Arg3)
 	} Else {
+		CreateDWordField(Arg3,0,CDW1)
```

Comment:
@piotr-kleins what are these changes for?

Reply:
ASL compiler gives warnings that CDW2 and CDW3 are unused, so I deleted them and made above part exactly the same way as it is in coreboot mainline

Reply:
@piotr-kleins the IASL compiler complained on it long time ago. Please revert back. We do not want weird behaviours on legacy.

Reply:
Done